### PR TITLE
Publish: 4-28-23, EOD; Whats new - AWS verified access

### PR DESCRIPTION
**Pull request by kiranhew**
AWS are planning to launch AWS Verified Access on **April 27, 2023.** 

This service allows IT teams to bypass VPN and ensure access to web applications is secure regardless of location. 

I need to add the 'LearnMoreLink' but it will be placed on explorer's hub and I don't have the URL yet. I can add it after.

### CODE REVIEW

File: src/content/whats-new/2023/04/whats-new-04-27-aws-verified-access.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 'New Relic supports AWS Verified Access'
+summary: 'Connect safely to your corporate applications without VPN and have a complete view of all your access logs'
+releaseDate: '2023-04-27'
+learnMoreLink: 'https://example.com'
```
Comment (akristen) [inline src/content/whats-new/2023/04/whats-new-04-27-aws-verified-access.md:5]: Is this meant to say example.com?

**Comment by akristen** [comment]
Check in with @kiranhew to confirm this is still good to go out on 4/27. Date might move!

**Comment by ally-sassman** [comment]
@akristen Sounds good, thank you!

**Comment by ally-sassman** [comment]
@jeff-colucci AWS is slated to release 4/28, but check in with Kiran to confirm before you merge this PR.

**Review by jeff-colucci — APPROVED** [review]
Looks good, once Gatsby gives the OK I'll get it merged!

**Pull request merged**